VPS/hosted e-mail guides

gary-ovh opened the pull request:
Updated servers on 1 hosted e-mail guide.
Created guide for enabling root user logins for VPS (Common ticket topic)
Added Qemu Agent installation information to automated backup guide

gary-ovh committed:
Hosted e-mails

Fixed wrong info

gary-ovh committed:
Merge remote-tracking branch 'upstream/develop' into develop

gary-ovh committed:
VPS root login guide

Added new guide
Stlll needs spanish and French Translations

gary-ovh committed:
Fixes

Fixed Warning spacing
Fixed Header, Excerpt, and slug

gary-ovh committed:
More fixes

Fixed restart SSH section

gary-ovh committed:
New guide / addition to existing guides

Added Qemu agent instructions to Automatic Backup guide
Added Qemu agent instructions to es-us version of snapshot guide
Added French version of enabling root password guide

gary-ovh committed:
Merge remote-tracking branch 'upstream/develop' into develop

bs-ovh committed:
Update guide.en-asia.md

Removing Snapshot section

bs-ovh committed:
Update guide.en-au.md

Removing Snapshot section

bs-ovh committed:
Update guide.en-ca.md

Removing Snapshot section

bs-ovh committed:
Update guide.en-sg.md

Removing Snapshot section

bs-ovh committed:
Update guide.en-us.md

Removing Snapshot section

bs-ovh committed:
Update guide.es-us.md

Removing Snapshot section

bs-ovh committed:
Update guide.fr-ca.md

Removing Snapshot section

bs-ovh committed:
Update guide.es-us.md

bs-ovh reviewed (approved):
Removed Snapshot section from Auto-Backup

davide-ovh-ca committed:
IP-LB additions and CephAAS updates

IP-LB additions, translations and missing web pages

CephAAS updates, screenshots and missing translations

davide-ovh-ca committed:
Revert "IP-LB additions and CephAAS updates"

This reverts commit 9bec5e9c9597a2ce286bbda89a63e3e04337c1af.

davide-ovh-ca committed:
Revert "Revert "IP-LB additions and CephAAS updates""

This reverts commit 409bab24b8908a2ffd5fc6c508a3ac17d6e52b40.

davide-ovh-ca committed:
Revert "Revert "Revert "IP-LB additions and CephAAS updates"""

This reverts commit 93a677ca6cd3f09abe8971a5dc9d55d229f86840.

davide-ovh-ca committed:
IP-LB additions and CephAAS updates 2

IP-LB additions, translations and missing web pages

CephAAS updates, screenshots and missing translations

gary-ovh committed:
Merge pull request #4 from davide-ovh-ca/develop

IP-LB additions and CephAAS updates 2

bs-ovh committed:
Update guide.en-ca.md

bs-ovh committed:
Update guide.en-asia.md

OVH-Team-Guides committed:
FR CA update automated backup for Cloud VPS

Adding QEMU agent

OVH-Team-Guides committed:
FR CA proofread Root login for Cloud VPS

OVH-Team-Guides commented:
Thanks a lot for your contribution @gary-ovh and @davide-ovh-ca ! Merge is done on develop, merge on master to happen soon
